src/linkml/include_participant.yaml

```diff
@@ -36,17 +36,23 @@ classes:
     - down_syndrome_status
     - ethnicity
     - external_id
-    - family_id
     - family_relationship
-    - family_type
-    - father_id
     - has_datafile
     - has_study
-    - mother_id
     - outcomes_vital_status
     - participant_id
     - race
     - sex
+  FamilyRelationship:
+    name: FamilyRelationship
+    definition_uri: include:FamilyRelationship
+    description: Object that captures relationships between the Participants and metadata about that relationship
```

Comment:
suggested definition:  "Object that capture a relationship between the Participant and one other member of their family, along with metadata about that relationship."